src/Processors/Formats/Impl/AvroRowInputFormat.cpp

```diff
@@ -68,17 +68,6 @@
 
 namespace DB
 {
-namespace ErrorCodes
```

Comment:
We should put only used ErrorCodes in .h and in .cpp files.

Reply:
pushed new commit as per your comments.